cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -1048,15 +1049,19 @@ private void startExecution(LoadMonitor loadMonitor,
    */
   private void sanityCheckOngoingMovement() throws OngoingExecutionException {
     boolean hasOngoingPartitionReassignments;
+    Map<TopicPartition, PartitionReassignment> ongoingPartitionReassignments;
     try {
-      hasOngoingPartitionReassignments = hasOngoingPartitionReassignments();
+       ongoingPartitionReassignments = ExecutionUtils.ongoingPartitionReassignments(_adminClient);
+      // avoid calling hasOngoingPartitionReassignments() to save 1 request.
+      hasOngoingPartitionReassignments = !ongoingPartitionReassignments.keySet().isEmpty();
     } catch (TimeoutException | InterruptedException | ExecutionException e) {
       // This may indicate transient (e.g. network) issues.
       throw new IllegalStateException("Failed to retrieve if there are already ongoing partition reassignments.", e);
     }
     // Note that in case there is an ongoing partition reassignment, we do not unpause metric sampling.
     if (hasOngoingPartitionReassignments) {
-      throw new OngoingExecutionException("There are ongoing inter-broker partition movements.");
+      LOG.error("Ongoing inter-broker partition movements: {}", ongoingPartitionReassignments);
+      throw new OngoingExecutionException("There are ongoing inter-broker partition movements");
```

Comment:
I think the error log here could be redundant. Ideally you only need this:

`throw new OngoingExecutionException("There are ongoing inter-broker partition movements: " + ongoingPartitionReassignments); `

And then the caller whoever handle that exception should be able to log error.

Reply:
My concern is if there's alot of partition movements the exception message will be pretty big which is not exactly desirable output. 
But if that's acceptable then i'm cool with that. 
Will make updates.

Reply:
That should be fine. It will be logged either ways.